package.json

```diff
@@ -3,5 +3,16 @@
     "devDependencies": {
         "less": "^2.7.2",
         "less-plugin-clean-css": "^1.5.1"
+    },
+    "dependencies": {
+        "opencollective": "^1.0.3"
+    },
+    "collective": {
+        "type": "opencollective",
+        "url": "https://opencollective.com/gitea",
+        "logo": "https://opencollective.com/gitea/logo.txt"
+    },
+    "scripts": {
+        "postinstall": "opencollective postinstall"
```

Comment:
Why do we need `opencollective` in the `package.json` since as I understand it, the avatars/placeholders in the `README.md` are static and do not require any JavaScript?

Reply:
If it isn't right for your flow, please delete /ignore it. The idea is that when users do an npm install, they would see "Like gitea? Become a backer/sponsor"